app/src/main/res/drawable/ic_launcher_foreground_primary_color.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Shouldn't it be possible to reuse the existing `ic_launcher_foreground.xml` and just apply a custom tint to it wherever this is needed?

Reply:
Sadly no. It's used in the Theme.App.Starting-Style for the new Splash screen, where you cant set a tint, afaik.

https://developer.android.com/develop/ui/views/launch/splash-screen#splash-screen-resources